src/main/java/DukeTest.java

```diff
@@ -0,0 +1,36 @@
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DukeTest {
+
+    /**
+     * Method to test the file reading capabilities of the storage class.
+     */
+    @Test
+    public void Test1() {
+        try {
+            String dir = System.getProperty("user.dir");
+            String textDir = dir + "/data/test1.txt";
+            Storage file = new Storage(textDir);
+            Todo test = new Todo("read book");
+            ArrayList<Task> arr = file.load();
+            assertEquals(arr.get(0).toString(), (new Todo("read book")).toString());
+        } catch (IOException e) {
+            System.out.println("Fail");
+        }
```

Comment:
Maybe you can test the loading functionality and the Todo Class functionality separately? It might make more sense to test Todo functionality first in this case separately, then test the loading.